sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/models/RuntimeVersion.java

```diff
@@ -1,6 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
This class is generated (you have `--preserve-model=UserSourceType,RuntimeVersion`)?

Reply:
It's generated in previous versions, they removed it from current version along with `UserSourceType`. I added them back so that it won't break our standard tier features.

Reply:
OK, then `--preserve-model` won't help. This flag is for model defined in swagger, but not referred in anywhere (method, parameter, other model, and hence removed by codegen, this flag tell codegen do not remove them).

Reply:
No, swagger does have `RuntimeVersion`?